doc/destination.md

```diff
@@ -0,0 +1,73 @@
+# Document Destinations
+
+Document destinations are the final location for a document transfer. When
+making a transfer request, the parameters you provide will depend on the source
+type.
+
+Unlike sources, the selected destination will also influence the response.
+
+_Note: Some destination types may require additional configuration of the
+service._
+
+## OneDrive
+
+With the OneDrive destination, the service will upload the document to a
+configured [Microsoft OneDrive][onedrive] account.
+
+### Configuration
+
+_**The OneDrive destination requires additional configuration of the service in
+order to function properly.**_
+
+Before you can use the OneDrive destination, you must create a new application
+in the Azure Portal, with read and write permissions to the appropriate OneDrive
+account. Created a client id and secret for the application to use for
+authenticating.
+
+The following environment variables must be set on the service:
+
+| Name                   | Description                                                         |
+|------------------------|---------------------------------------------------------------------|
+| ONEDRIVE_CLIENT_ID     | The client ID of the OneDrive application.                          |
```

Comment:
Suggestion: change description to `The application (client) ID of the OneDrive application` to match the app registration nomenclature